File: bot.py
import os
import discord

from discord.ext import commands 

intents = discord.Intents.default()

with open("token.txt", "rb") as file:
	TOKEN = file.read().decode('utf-8')
print(TOKEN)

bot = discord.Client(intents = intents)

@bot.event
async def on_ready():
	print('Bot is online')

@bot.event
async def on_message(message):
	if (message.content == "!ping"):
		print('chuj lol')

bot.run(TOKEN)

File: bot.js
const { Client } = require('discord.js');
const functions = require('./forExport.js');
const commands = require('./commandHandler.js');
const fs = require('fs');

let token = fs.readFileSync('token.txt', 'utf8');
const bot = new Client();

const BOT_TOKEN = token;
bot.login(BOT_TOKEN);

bot.on('ready', () => {
  console.log(`Logged in as ${bot.user.tag}!`);
});

bot.on('message', message => {
	commands.commandHandler(message);
});